app/src/main/java/fr/free/nrw/commons/repository/ProfileRepository.kt

```diff
@@ -0,0 +1,53 @@
+package fr.free.nrw.commons.repository
+
+import fr.free.nrw.commons.network.APIService
+import fr.free.nrw.commons.profile.achievements.LevelController
+import fr.free.nrw.commons.profile.model.UserAchievements
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.flow
+import timber.log.Timber
+import javax.inject.Inject
+
+class ProfileRepository @Inject constructor(private val apiService: APIService) {
```

Comment:
Question:
Should this be AchievementRepository? or we can have profile repository which will be a repository layer for Achievement, Leaderboard,  Contribution.?